src/tips/management/commands/reset_tips_histories.py

```diff
@@ -0,0 +1,43 @@
+from django.core.management.base import BaseCommand, CommandError
+
+from django.contrib.auth.models import User
+from tips.models import TipHistory
+
+class Command(BaseCommand):
+    help = 'In the tip history table, reset these field values:\n' \
+           '1. reset the show count to 0\n' \
+           '2. reset all timestamps to null including the last show, the last attempt and the last action\n' \
+           'If a user name is given as an argument, above values for this user will be reset.\n' \
+           'If no argument is given, all above values in the tip history table will be reset.\n' \
+           '\n'
+
+    def add_arguments(self, parser):
+        parser.add_argument('username', nargs='?', type=str)
+
+    def handle(self, *args, **options):
+        uname = options['username']
+        if uname:
+            user = User.objects.filter(username=uname)
```

Comment:
I see you ended up using the double underscore technique to thread through the models.  That's better than what I suggested since it gets the ClusiveUser instance in one line.